### ISSUE TITLE
Results in json format Map<String, dynamic>

### ISSUE THREAD

**Issue by Akash98Sky** [Kerego/tmdb_dart#13]
@Kerego I was thinking if we save the jsons as Map, in the models and use getters for each of the Json keys. 

In this way anyone can use the Maps only instead of the predefined class formats if they wish to.

**Comment by Kerego** [comment]
Hi @Akash98Sky, 

I am probably missing a point here, but I'm not sure what is the use case of this. 
The original idea of the library was to transform some of the oddities in the TMDB API's data into something more useful and easier to use from an app developer perspective.
Using Maps directly would be kinda equivalent to doing HTTP requests directly without using the library at all.

**Comment by Akash98Sky** [comment]
Yeah it's fine. But if we add Map with getters for its keys. Then others can just use it as they wish. I mean it would be a good way to get json formats of data. If anyone wishes to use. And also the objects if needed.

**Comment by Kerego** [comment]
Ok, now I see, this could be helpful in offline scenarios when the app needs to save some amount of data locally and show it if there is no internet connection or when the API doesn't provide a way to access multiple movie details in a single request (something like favorites, or watchlist).
I think this would be a great addition to the library, great idea @Akash98Sky.
